mava/utils/lp_utils.py

```diff
@@ -88,17 +88,15 @@ def partial_kwargs(function: Callable[..., Any], **kwargs: Any) -> Callable[...,
     return functools.partial(function, **kwargs)
 
 
-def termination_fn(
-    parameter_server: SystemParameterServer,
-) -> None:
+def termination_fn(node: Union[SystemParameterServer, SystemExecutor]) -> None:
     """Terminate the process
 
     Args:
-        parameter_server: SystemParameterServer in order to get main pid
+        node: SystemParameterServer in order to get main pid
```

Comment:
Thanks @OmaymaMahjoub :slightly_smiling_face: Just add here that this can be a SystemExecutor as well.

Reply:
Just remove the SystemExecutor now :sweat_smile: We are always killing in the parameter server now right?